@kiva/kv-components/vue/KvChip.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+	<div class="tw-inline-flex">
+		<button
+			class="
+			chip
+			tw-bg-tertiary
+			tw-items-center
+			tw-flex
+			tw-flex-row
+			tw-whitespace-nowrap
+			"
+			@click="handleClick"
+		>
+			<div class="tw-text-small">
+				{{ title }}
+			</div>
+			<div class="close-button-container tw-text-right">
+				<kv-material-icon
+					name="small-x"
+					class="close-button tw-fill-gray-600"
+					:from-sprite="true"
+				/>
+			</div>
+		</button>
+	</div>
+</template>
+<script>
+import KvMaterialIcon from './KvMaterialIcon.vue';
+
+export default {
+	name: 'KvChip',
+	components: { KvMaterialIcon },
+	props: {
+		title: {
+			type: String,
+			required: true,
+		},
+	},
+	methods: {
+		handleClick() {
+			this.$emit('click-chip');
+		},
+	},
+};
+</script>
+
+<style lang="scss" scoped>
+
+	.chip {
+		border-radius: rem-calc(4);
```

Comment:
How many of these styles can be replaced with tailwind equivalents?

Reply:
@dyersituations  Likely will change convert line 64- I went through already and converted most. Unsure on the rest since the Tailwinds primitive classes caps at 4 rem

Reply:
My suggestion here is to use the "closest" available value from tailwind. It may look slightly different than the design but you can just make sure the label is balanced and looks good within the chip using our global sizes and it will be good.